workspaces/rbac/plugins/rbac-backend/src/policies/permission-policy.test.ts

```diff
@@ -206,13 +211,13 @@ describe('RBACPermissionPolicy Tests', () => {
     });
 
     // case3
-    it('should allow deny access to resource permission for user:default/known_user', async () => {
+    it('should alloshould allow read access to resource permission for user from csv filew deny access to resource permission for user:default/known_user', async () => {
```

Comment:
Fix typo, please. I think you wanted to provide correct test title: `should allow read access to resource permission for user from csv file`